debezium-core/src/main/java/io/debezium/pipeline/ChangeEventSourceCoordinator.java

```diff
@@ -193,11 +196,37 @@ protected void executeChangeEventSources(CdcSourceTaskContext taskContext, Snaps
         LOGGER.debug("Snapshot result {}", snapshotResult);
 
         if (running && snapshotResult.isCompletedOrSkipped()) {
+            if (snapshotResult.isCompleted()) {
```

Comment:
Why do delay only in case of  `completed` snapshot? I know that the goal here is to let the connector to flush offsets and avoid a re-snapshot on restart but I think this should be more general delay.

Reply:
If the connector is configured to skip snapshot, then we don't want to add an unnecessary delay for streaming as there is no snapshot offset to be committed.
Also, for aborted/failed snapshots, the streaming won't start anyway, so no need to add a delay.

Reply:
I just saw that you mentioned the behavior in the field description, so I am fine with that. Please add it also on the documentation.